ext/tokenizer/tokenizer_data.c

```diff
@@ -16,7 +16,7 @@
 
 /*
    DO NOT EDIT THIS FILE!
-   This file is generated using tokenizer_data_gen.php
+   This file is generated using tokenizer_data_gen.sh
```

Comment:
The commits in this PR seems like they should be rebased and squashed - the changes to tokenizer_data.c ended up being unnecessary?